assets/src/edit-story/components/panels/index.js

```diff
@@ -57,6 +58,7 @@ const SIZE_POSITION = 'sizePosition';
 const SHAPE_STYLE = 'shapeStyle';
 const TEXT = 'text';
 const TEXT_STYLE = 'textStyle';
+const VIDEO_POSTER = 'poster';
```

Comment:
Nit:

There's `VIDEO_OPTIONS = 'videoOptions'`, `VIDEO_ACCESSIBILITY = 'videoAccessibility'`, ... why not `VIDEO_POSTER = 'videoPoster'`?

Reply:
Updated to 'videoPoster'.